packages/cali/src/cli.ts

```diff
@@ -88,7 +82,7 @@ while (true) {
   s.start(chalk.gray('Thinking...'))
 
   const response = await generateText({
-    model: openai(AI_MODEL),
+    model,
```

Comment:
Thanks for working on this!

This will probably clash with @grabbou PR: https://github.com/callstackincubator/cali/pull/57

Let's get the other one merged first and then this one